Added classes for DisplacementSensor and HeadingSensor

vincent-miller opened the pull request:
Added two more empty classes in which to implement reading information from the mouse and heading sensor. This data will be integrated in PositionTracking.

celskeggs commented:
I'll merge this once it looks good, but it's probably not worth adding pull requests for small additions if no one else is depending on them to do their work.

celskeggs commented on QuasarHelios2015/src/org/team1540/quasarhelios/DisplacementSensor.java:
```diff
@@ -0,0 +1,13 @@
+package org.team1540.quasarhelios;
+
+import java.awt.geom.Point2D;
```
Please don't use AWT unless you're doing graphics in your code somewhere. I'd rather you write your own Point2D than depend on functionality from Java's graphics library.

celskeggs commented:
To clarify, you should still send everything through code review, but don't send it until you have something significant to be reviewed.

vincent-miller commented:
I've removed the AWT dependencies and implemented basic integration. The integration method may have to be improved (for example if we want to include the headings at the beginning and end of a time interval).

vincent-miller commented:
Should I change PositionTracking.getPosition to use CCRE FloatInputs instead (replace it with getXChannel and getYChannel)?

vincent-miller commented:
When I poll the mouse, the return is displacement since the last event, not velocity. This means that it is time-independent. The getDisplacement method would block until a new mouse event is available (unless you think it shouldn't be) so that would serve in place of a time delay.

celskeggs commented:
Changing getPosition to use that sounds useful.

That makes more sense with `getDisplacement`. Perhaps you should rename it to make it clear what it does? (And perhaps reorder them so that it gets the heading after each displacement, not before.)

celskeggs commented:
We've canceled this. Oh well.
